packages/ods/src/components/link/src/components/ods-link/ods-link.tsx

```diff
@@ -0,0 +1,48 @@
+import { Component, type FunctionalComponent, Host, Prop, h } from '@stencil/core';
+import { type OdsIconName } from '../../../../icon/src/constants/icon-name';
+import { ODS_LINK_COLOR, type OdsLinkColor } from '../../constant/link-color';
+
+@Component({
+  shadow: true,
+  styleUrl: 'ods-link.scss',
+  tag: 'ods-link',
+})
+export class OdsLink {
+  @Prop({ reflect: true }) public color: OdsLinkColor = ODS_LINK_COLOR.primary;
+  @Prop({ reflect: true }) public disabled?: boolean;
+  @Prop({ reflect: true }) public download?: HTMLAnchorElement['download'];
+  @Prop({ reflect: true }) public href!: string;
+  @Prop({ reflect: true }) public icon?: OdsIconName;
+  @Prop({ reflect: true }) public label?: string;
+  @Prop({ reflect: true }) public referrerpolicy?: ReferrerPolicy;
+  @Prop({ reflect: true }) public rel?: HTMLAnchorElement['rel'];
+  @Prop({ reflect: true }) public target?: HTMLAnchorElement['target'];
+
+  render(): FunctionalComponent {
+    return (
+      <Host class='ods-link'>
+        <a
+          class={{
+            'ods-link__link': true,
+            'ods-link__link--disabled': this.disabled ?? false,
+            [`ods-link__link--${this.color}`]: true,
+          }}
+          download={ this.download }
+          href={ this.href }
+          part="link"
+          referrerPolicy={ this.referrerpolicy }
+          rel={ this.rel }
+          tabindex={ this.disabled ? -1 : 0 }
```

Comment:
shouldn't be needed

Reply:
without that the link is navigable when disabled